### PR TITLE
User book correlation

### PR DESCRIPTION
**Pull request by nowicki-maciej**

### CODE REVIEW

File: flavors/web-app/src/main/java/cf/dmms/app/web/api/book/BookUploadController.java [hunk 1]
```diff
@@ -42,6 +43,7 @@ public BookUploadController(BookService bookService, BookMapper bookMapper, Obje
 
 	@PostMapping
```
Comment (michal-krajewski) [inline flavors/web-app/src/main/java/cf/dmms/app/web/api/book/BookUploadController.java:44]: Please specify endpoint value

File: flavors/web-app/src/main/java/cf/dmms/app/web/api/book/BookUploadController.java [hunk 2]
```diff
@@ -42,6 +43,7 @@ public BookUploadController(BookService bookService, BookMapper bookMapper, Obje
...
 	public BookDto addBook(
```
Comment (michal-krajewski) [inline flavors/web-app/src/main/java/cf/dmms/app/web/api/book/BookUploadController.java:45]: I think that this endpoint could be just in book controller. Consider moving part of methods to BookService or create separate service for uploading

File: user-management/src/main/java/cf/dmms/app/usermanagement/SecurityConfig.java [hunk 3]
```diff
@@ -43,24 +41,26 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
 	protected void configure(HttpSecurity http) throws Exception {
 		http
 				.csrf()
-					.csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse())
-					.and()
+				.csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse())
```
Comment (michal-krajewski) [inline user-management/src/main/java/cf/dmms/app/usermanagement/SecurityConfig.java:44]: Please fix formatting.

File: spi/src/main/java/cf/dmms/app/spi/Book.java [hunk 4]
```diff
@@ -1,7 +1,6 @@
-package cf.dmms.app.core.book;
+package cf.dmms.app.spi;
```
Comment (michal-krajewski) [inline spi/src/main/java/cf/dmms/app/spi/Book.java:1]: Why did you move Book from core to spi?

Comment (michal-krajewski) [reply spi/src/main/java/cf/dmms/app/spi/Book.java:1]: I thought that only user was supposed to be moved, and then core would include spi

**Review by michal-krajewski — CHANGES_REQUESTED** [review]

File: spi/src/main/java/cf/dmms/app/spi/Book.java [hunk 5]
```diff
@@ -15,6 +14,10 @@
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Long id;
 
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name="user_id")
+	private User user;
```
Comment (michal-krajewski) [inline spi/src/main/java/cf/dmms/app/spi/Book.java:19]: Maybe "owner" instead of "user"? I think that would be more appropriate name.

File: spi/src/main/java/cf/dmms/app/spi/Book.java (hunk 4 above)
Comment (nowicki-maciej) [reply spi/src/main/java/cf/dmms/app/spi/Book.java:1]: core does include spi, but having entities all around is not a good idea in my opinion. Also, if user is in SPI why book is not? Some kind of logic problem here I suppose
